process/transaction/shardProcess.go

```diff
@@ -236,7 +236,7 @@ func (txProc *txProcessor) executingFailedTransaction(
 		return nil
 	}
 
-	txFee := txProc.economicsFee.ComputeFee(tx)
+	txFee := core.SafeMul(tx.GasPrice, tx.GasLimit)
```

Comment:
So, when execution will fail we will take all the gas set? Could be a situation when GasLimit is set to 0 and we will not take anything? Is this check, for min gas limit, done on interceptor for every type of transaction?

Reply:
There is no scenario when gasLimit is set to 0 - it will not pass interceptor.

Reply:
ok